site/content/docs/Encryption/_index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Encryption"
+linkTitle: "Encryption"
+weight: 4
+---
+
+Kopia uses a standard envelope encryption technique to de-couple the repository passphrase from the keys used for encrypting and authenticating the contents of the repository.
+
+Each repository has a format blob containing configuration parameters for the repository. The format blob itself is serialized and persistent as JSON, such as the one below.
+
+not a good random initial value
```

Comment:
not sure what `not a good random inital value` is?

Reply:
uh? me neither.
```suggestion
```

Reply:
Oh, now I remember => `bm90IGEgZ29vZCByYW5kb20gaW5pdGlhbCB2YWx1ZQ==`
And it was only an hour ago 😞